### PR TITLE
CLOUDP-173973: Improve error handling for cluster describe if no output and name is provided

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

_Jira ticket:_ CLOUDP-173973

Improve validation of input for clusters given certain conditions will trigger different endpoints with different types

### CODE REVIEW

File: internal/cli/opsmanager/clusters/describe.go [hunk 1]
```diff
@@ -55,6 +56,14 @@ func (opts *DescribeOpts) Run() error {
 
 	return opts.Print(r)
 }
+func (opts *DescribeOpts) validateArg() error {
```
Comment (blva) [inline internal/cli/opsmanager/clusters/describe.go:59]: this got me a bit confused, when we list clusters, we get ID and name

```shell
~ ❯ atlas clusters list                                             7s 13:50:52
ID                         NAME           MDB VER   STATE
644a6faa12c63d0ea45ef37a   Cluster99845   6.0.5     CREATING
```

Are we supposed to validate the name as ID? maybe it should be another flag?

Comment (blva) [reply internal/cli/opsmanager/clusters/describe.go:59]: sorry wait I used altas here let me check OM

Comment (gssbzn) [reply internal/cli/opsmanager/clusters/describe.go:59]: >maybe it should be another flag?
 
```
mongocli ops-manager clusters describe --help
When describing cluster with no output format please provide the cluster ID.
When using an output format the please provide the cluster name.
```


At this point any change would be a breaking change there's a ticket to figure out what to do with these commands but it hasn't been prioritized so far

Comment (blva) [reply internal/cli/opsmanager/clusters/describe.go:59]: ah, got it! thanks for the clarification

File: internal/cli/opsmanager/clusters/describe.go [hunk 2]
```diff
@@ -55,6 +56,14 @@ func (opts *DescribeOpts) Run() error {
...
+	if opts.ConfigOutput() == "" {
+		if err := validate.ObjectID(opts.name); err != nil {
+			return fmt.Errorf("please provide a valid cluster ID, %w", err)
```
Comment (blva) [inline internal/cli/opsmanager/clusters/describe.go:62]:
```suggestion
			return fmt.Errorf("please provide a valid cluster ID or provide a cluster name and output format, %w", err)
```

**Review by blva — APPROVED** [review]
LGTM

**Review by blva — APPROVED** [review]

**Pull request merged**